emails/views.py

```diff
@@ -1414,12 +1444,12 @@ def _get_domain_address(local_portion: str, domain_portion: str) -> DomainAddres
         raise e
 
 
-def _get_address(address: str) -> RelayAddress | DomainAddress:
+def _get_address(address: str, create: bool = True) -> RelayAddress | DomainAddress:
     """
     Find or create the RelayAddress or DomainAddress for an email address.
 
-    If an unknown email address is for a valid subdomain, a new DomainAddress
-    will be created.
+    If an unknown email address is for a valid subdomain, and create is True,
+    a new DomainAddress will be created.
```

Comment:
(question, blocking) Can you explain why this was added? When will the `create` be set to `False`?

(suggestion, blocking) Also, if the `create` parameter is only used to get/create `DomainAddress` logic, I recommend making the `create` parameter more descriptive like `create_domainaddress`.